app/src/components/mostTradedTokenTransferCard.tsx

```diff
@@ -40,14 +40,14 @@ export const MostTradedTokenTransferCard = (props: MostTradedTokenTransferCardPr
 
   return (
     <React.Fragment>
-      { !error && (!props.tokenTransfers || isLoading || !asset || !collection) ? (
+      { !error && (!props.tokenTransfers || isLoading) ? (
```

Comment:
right i see whats going wrong. you're right, lets have an errorcard and undo these changes.

Reply:
and then remvoe error from NftCard i think.